client/src/styles/trumps/icon.fonts.css

```diff
@@ -2,7 +2,7 @@
   font-family: 'Cortal-Icons';
   /* TACC: Use `woff` because it is `ttf` optimized for web, and browser support */
   /* TACC: Renamed to include version and correct path */
-  src: url('../../fonts/Cortal-Icons-1.2.woff?vrusnv') format('woff');
```

Comment:
You may also delete this file. It is not loaded elsewhere by Portal (nor CMS nor Docs). Can be done via [FP-1394](https://jira.tacc.utexas.edu/browse/FP-1394) instead.